lib/repository.py

```diff
@@ -35,12 +36,16 @@ def __init__(self, push_info):
         super(PushError, self).__init__(message)
 
 
-def get_git_repository(name, remote_repo_url, parent_dir_path):
+def get_git_repository(remote_repo_url, parent_dir_path):
     """
-    Get a local git repository located in a subdirectory of the parent
-    dir, according to specified name.
+    Get a local git repository located in a subdirectory of the parent directory,
+    named after the file name of the URL path (git default).
     If it does not exist, clone it from the specified URL.
     """
+    # infer name
```

Comment:
IMHO considering a comment to be obvious (and thus useless) or not is usually a matter of personal taste. Very minor issue IMHO. Anyway, I have updated the comment to make it more useful.